test/docs/discriminators.test.js

```diff
@@ -107,9 +107,9 @@ describe('discriminator docs', function() {
     event.__t = 'SignedUp';
     // ValidationError: ClickedLink validation failed: __t: Cast to String failed for value "SignedUp" (type string) at path "__t"
     // acquit:ignore:start
-    await assert.rejects(async () => {
+    await assert.rejects(async() => {
     // acquit:ignore:end
-    await event.save();
+      await event.save();
```

Comment:
Is there any way we can avoid this one indentation change? The indentation is slightly off here because the `await assert.rejects()` is stripped from the output because of `// acquit:ignore:start`, so we want the docs to appear as if `await event.save()` isn't indented at all.

Reply:
we could disable indentation rule for that line, but would not recommend to to it globally

or the acquit (plugin?) could handle indentation changes between comments

Reply:
for now i could add a quick-fix of ignoring the eslint rule (but editors may still automatically apply indentation)

or we could wait for https://github.com/vkarpov15/acquit-ignore/pull/6